styleguide/derek/view-templates/search/search.ejs

```diff
@@ -0,0 +1,54 @@
+<!--Header Medium-->
+<div class="imacBanner">
+  <div class="imacBanner-container">
+    <div class="imacBanner-textRow">
+      <div class="imacBanner-textContainer imacBanner-headlineMedium">
+        <h2 class="imacBanner-headline">Search Results</h2>
+        <span class="imacBanner-line left-top"></span>
+        <span class="imacBanner-line left-bottom"></span>
+        <span class="imacBanner-line right-bottom"></span>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="container half-padding">
+  <!-- Lead pattern -->
+  <div class="lead-wrapper">
+    <div class="lead-headerWrapper text-left">
+      <h1 class="lead-header">Search Results for: Cloud</h1>
```

Comment:
@pobtastic  I showed this to the CROUX team yesterday and everyone gave a +1 as is, since its using our current styles. It was discussed adding more at a later date.